plugins/excel/ui/src/test/java/org/pentaho/di/ui/trans/steps/excelinput/ExcelInputDialogTest.java

```diff
@@ -0,0 +1,94 @@
+///*! ******************************************************************************
+// *
+// * Pentaho Data Integration
```

Comment:
Why is this entire test commented out?

Reply:
Commented while developing and by mistake forgot to uncomment while pushing changes